Documentation/gettingstarted/requirements-rke.rst

```diff
@@ -0,0 +1,28 @@
+**Default Configuration:**
+
+=============== =============== ==============
+Datapath        IPAM            Datastore
+=============== =============== ==============
+Encapsulation   Cluster Pool    Kubernetes CRD
+=============== =============== ==============
+
+**Requirements:**
+
+* Follow the `RKE Installation Guide <https://rancher.com/docs/rke/latest/en/installation/>`_
```

Comment:
I think this needs updating to https://docs.rke2.io/install/network_options/#using-cilium-instead-of-canal (cc @seanmwinn)